### PR TITLE
Add a feed where all posts of all circles are curated

### LINKED ISSUE #150: As a user, I want to have lists of posts, so that I can view the feed of current posts
Create a "Home" feed

Related to #18

### PR DESCRIPTION
**Pull request by metalmatze**
This creates a feed for all posts in all circles. :balloon: :tada: 
Closes #150 

download 9

### CODE REVIEW

File: circuit/urls.py [hunk 1]
```diff
@@ -26,4 +26,5 @@
     url(r'^$', views.LandingPage.as_view(), name='landingpage'),
     url(r'^circles/', include('circles.urls', namespace='circles')),
     url(r'^legal-notice/', views.LegalNotice.as_view(), name='legal-notice'),
+    url(r'^feed/', views.feed, name='feed')
```
Comment (juanecabellob) [inline circuit/urls.py:30]: here it was supposed to be the root, wasn't it?

Comment (Gjum) [inline circuit/urls.py:30]: Has to be changed when the landingpage is done.

Comment (metalmatze) [inline circuit/urls.py:30]: Exactly.

**Comment by Gjum** [comment]
Fancy. Ima merge this.

**Comment by Gjum** [comment]
:P

**Comment by Gjum** [comment]
merged manually ... somehow

**Comment by juanecabellob** [comment]
image

**Pull request closed without merging**